app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivityViewModel.java

```diff
@@ -35,4 +40,39 @@ public void setFrom(Place from) {
         this.from = from;
     }
 
-}
+    public DirectionsResult getDirectionsResult() {
+        return directionsResult;
+    }
+
+    public void setDirectionsResult(DirectionsResult directionsResult) {
+        this.directionsResult = directionsResult;
+    }
+
+    /**
+     * Build the URL that will be used to call the Google Maps Directions API by passing the necessary parameters
+     *
+     * @param from: latitude and longitude of the origin
+     * @param to: latitude and longitude of the destination
+     * @param transportType: main transport type to get from origin to destination
+     *
+     */
+    public String buildUrl(LatLng from, LatLng to, @ClassConstants.TransportType String transportType) {
+        // Origin of route
+        String str_origin = "origin=" + from.latitude + "," + from.longitude;
```

Comment:
I think all these comments are useless, the naming of your variables is good enough

Reply:
you're right!